Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/CommandLine.kt

```diff
@@ -103,6 +104,16 @@ class CInteropArguments(argParser: ArgParser =
     val linkerOption = argParser.option(ArgType.String, "linker-option",
             description = "additional linker option").multiple()
     val linker by argParser.option(ArgType.String, description = "use specified linker")
+
+    val compileSource by argParser.option(ArgType.String,
+            fullName = "Xcompile-source",
+            description = "additional C/C++ sources to be compiled into resulting library"
+    ).multiple().delimiter(",")
+
+    val sourceCompileOptions by argParser.option(ArgType.String,
+            fullName = "Xsource-compile-options",
```

Comment:
`Xsource-compiler-option` would be more consistent with existing flags, esp. `-compiler-option`.

Reply:
This would also require removing `.delimiter(",")`.